src/api/utils/reports.py

```diff
@@ -0,0 +1,39 @@
+from ..mocks import MockRequest
+from ..models import (
+    BENTHICLIT_PROTOCOL,
```

Comment:
I think we should add modifying this to the README documenting what needs to be done to add a new protocol